src/main/java/de/metas/ui/web/request/process/WEBUI_CreateRequest.java

```diff
@@ -62,9 +66,16 @@ protected String doIt() throws Exception
 		final String tableName = getTableName();
 		if (I_C_BPartner.Table_Name.equals(tableName))
 		{
-			final I_C_BPartner bpartner = getRecord(I_C_BPartner.class);
+			final IBPartnerDAO bPartnerDAO = Services.get(IBPartnerDAO.class);
```

Comment:
pls declare it as private final fields in the process class. (see background further down)